### ISSUE TITLE
Why asking the budget twice ? 

### ISSUE THREAD

**Issue by KartikeyBartwal** [Open-Source-Chandigarh/Expense-Tracker#9]
Can we ask the budget only at 1 place ? The index page's budget doesn't appear to be at use anywhere for the user main experience.
![image](https://github.com/Open-Source-Chandigarh/Expense-Tracker/assets/98084657/3837a25d-039a-4611-b5a5-44af566a42df)
....
![image](https://github.com/Open-Source-Chandigarh/Expense-Tracker/assets/98084657/508c5ea4-3412-463e-b6e1-5e39628a157e)

**Comment by BamaCharanChhandogi** [comment]
Yes, this is also a problem. I think we should work on this. You can start the design of the page.

**Comment by KartikeyBartwal** [comment]
Do you want to keep the index page ? Kindly share your vision on the index page and the PR overall.

**Comment by BamaCharanChhandogi** [comment]
You can keep the index page right now. Users are entering their wallet values on the index page. You can set that value to the main page balance. You can remove the alert section.

**Comment by KartikeyBartwal** [comment]
Really sorry but I won't be able to complete this issue due to my current tight schedule 🙏
If someone else wants to work on this, please message

**Comment by BowlPulp** [comment]
I'd love to contribute to this, kindly allow me to fix this issue!

**Comment by KartikeyBartwal** [comment]
> I'd love to contribute to this, kindly allow me to fix this issue!

That's great!
Would it be acceptable to you, @BamaCharanChhandogi?
